README.md

```diff
@@ -167,6 +170,18 @@ See [`examples/custom-tool-grader-and-extractor/`](examples/custom-tool-grader-a
 
 * Yes! Use `--cached-results` to reuse agent trajectories across evaluations, avoiding redundant agent runs when testing different graders.
 
+**Can I evaluate Letta Code agents across different models?**
+
+* Yes! The Letta Code target now supports multi-model evaluation. In your suite YAML, specify multiple model handles:
```

Comment:
nit: can we do`evaluating multiple models` instead of `multi-model`? I think `multi-model` is slightly confusing to me (eg. might mean routing betwen multiple models or something)